credit_test.go

```diff
@@ -0,0 +1,290 @@
+// Copyright 2020 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package imagecashletter
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+)
+
+// mockCredit creates a Credit
+func mockCredit() *Credit {
```

Comment:
This should either parse from a string like line 18, or set each field manually like 23-33. Setting the fields manually is more consistent with other records, so I'd suggest this approach: https://github.com/moov-io/imagecashletter/blob/3c6d85e84784de5ddb08cebb2bdf4f6cbbec83d0/creditItem_test.go#L13-L26